plugin/src/software/aws/toolkits/eclipse/amazonq/util/Constants.java

```diff
@@ -19,6 +19,9 @@ private Constants() {
     public static final String LSP_CODE_REFERENCES_OPT_OUT_KEY = "includeSuggestionsWithCodeReferences";
     public static final String IDE_CUSTOMIZATION_NOTIFICATION_TITLE = "Amazon Q Customization";
     public static final String IDE_CUSTOMIZATION_NOTIFICATION_BODY_TEMPLATE = "Amazon Q inline suggestions are now coming from the %s";
+    public static final String MANIFEST_DEPRECATED_NOTIFICATION_TITLE = "Amazon Q Manifest Version is Deprecated";
```

Comment:
To a user "manifest version" does not mean anything. It is an implementation detail. "Update Amazon Q extension" 
whatever is the official name of the extension on marketplace?

Reply:
Language updated